news/change-1270.md

```diff
@@ -0,0 +1 @@
+Rename yaml config file `IsIdentifiableBaseOptions` to `IsIdentifiableOptions` and removed unused CLI verbs in `smi`
```

Comment:
Can you rename this file so it matches the format needed for the changelog script? Should be `<pr#>-<type>.md` as specified in `news/README.md`.